### PR TITLE
Lms 1626 replay single events

### PR DESCRIPTION
**Pull request by lzabo**
**Description**
add css to report page.
add Ajax script to move back any number of event what a developer set.
add javascript using amd to handle an individual request with ajax.
add some extra string to the lang file.

**Related Issues**
No related issue.

**PR Type**
Enhancement

### CODE REVIEW

File: amd/src/replayevents.js [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+            },
+
+            /**
+             * Generate done icon.
+             */
+            generateDoneHTML : function(){
+                str.get_strings([
+                    {
+                        key: 'success',
+                        component: 'moodle'
```
Comment (lzabo) [inline amd/src/replayevents.js:366]: I will set it to come from the logstore_xapi module.

File: amd/src/replayevents.js [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+            },
+
+            /**
+             * Generate load icon.
+             */
+            generateLoadHTML : function(){
+                str.get_strings([
+                    {
+                        key: 'loadinghelp',
+                        component: 'moodle'
```
Comment (lzabo) [inline amd/src/replayevents.js:349]: I will set it to come from the logstore_xapi module.

File: report.php [hunk 3]
```diff
@@ -112,7 +105,9 @@
     }
     $table->head[] = get_string('info', 'logstore_xapi');
     $table->head[] = get_string('datetimegmt', 'logstore_xapi');
+    $table->head[] = '';
     $table->id = "report";
+    $replayevent = get_string('replayevent', 'logstore_xapi');
```
Comment (stephen-oh) [inline report.php]: Is this variable used?

Comment (lzabo) [reply report.php]: Nope, good catch. It was moved to the other place. I will remove it.

**Review by gordonmacqueen-lp — APPROVED** [review]

**Pull request merged**